litestar/logging/config.py

```diff
@@ -313,7 +314,9 @@ def stdlib_json_serializer(value: EventDict, **_: Any) -> str:  # pragma: no cov
     return _msgspec_json_encoder.encode(value).decode("utf-8")
 
 
-def default_structlog_processors(as_json: bool = True) -> list[Processor]:  # pyright: ignore
+def default_structlog_processors(
+    as_json: bool = True, json_serializer: Callable[[Any], Any] = default_json_serializer
+) -> list[Processor]:  # pyright: ignore
```

Comment:
To use the `WriteLoggerFactory` with a file, the processor chain needs to return a string. This just gives me a way to pass the json encoder in without needing to build another processor chain.